packs/classfeatures/breakthrough-innovation.json

```diff
@@ -11,7 +11,7 @@
 which it was already an expert (because of being an advanced construct companion, for example), it gains master proficiency instead. If you have the revolutionary innovation class feature, these proficiencies improve to master, or legendary if your construct innovation was already an expert. You must have the wonder gears modification to select this modification.</li>\n<li><strong>Turret Configuration</strong> Your innovation can transform from a mobile construct to a stationary turret. Your construct companion can transform as a single action, which has the manipulate trait, turning into a turret in its space (or transforming back from a turret into its normal configuration). While it's a turret, your innovation is @UUID[Compendium.pf2e.conditionitems.Item.Immobilized], but the damage die from its projectile launcher increases to 1d6 and the range increment increases to 60 feet. You must have the projectile launcher modification to select this modification.</li>\n</ul>\n<h3>Breakthrough Weapon Modifications</h3>\n<ul>\n<li><strong>Advanced Rangefinder (Ranged Only)</strong> A carefully tuned scope or targeting device makes your weapon especially good at hitting weak points. Your innovation gains the backstabber trait and increases its range increment by 10 feet.</li>\n<li><strong>Aerodynamic Construction (Melee Only)</strong> You carefully engineer the shape of your weapon to maintain its momentum in attacks against successive targets. Your innovation gains the sweep trait and the versatile S trait.</li>\n<li><strong>Inconspicuous Appearance (Melee Only)</strong> Your innovation is built for easy concealment and surprise attacks. It gains the backstabber and versatile P traits. If the weapon has light Bulk, it also gains the concealable trait.</li>\n<li><strong>Integrated Gauntlet (One-Handed Weapon Only; Can't Have the Two-Hand Trait)</strong> Combining your weapon with a gauntlet, you make it so you can quickly switch between attacking with your weapon and tinkering using your hands. Your innovation gains the free-hand trait.</li>\n<li><strong>Manifold Alloy</strong> You've devised several alloys that contain useful properties of various metals, and you replace a number of parts of your weapon or its ammunition with pieces from these alloys. Your weapon innovation is cold iron and silver. This means you deal more damage to a variety of supernatural creatures, such as demons, devils, and fey.</li>\n<li><strong>Rope Shot (Ranged Only)</strong> Your weapon can shoot projectiles that split into simple ropes or nets around your foes' legs to trip your targets, and you can climb using the grappling hooks built into the weapon. Your weapon innovation gains the climbing and ranged trip traits.</li>\n<li><strong>Tangle Line (Thrown Only)</strong> Your weapon has an extensible line that you can use to knock your enemies over and quickly recall the weapon back to your hand. Your innovation gains the ranged trip trait and the tethered trait.</li>\n</ul>"
```

Comment:
mention of "strength score"

Reply:
Fixed the references - replaced score with modifier and the appropriate attribute value.